nautobot/utilities/testing/api.py

```diff
@@ -256,6 +256,32 @@ def test_options_returns_expected_choices(self):
 
             self.assertEqual(set(self.choices_fields), field_choices)
 
+        @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
+        def test_status_options_returns_expected_choices(self):
+            # Set to self.choices_fields as empty set to compare classes that shouldn't have any choice fields on serializer.
+            if not self.choices_fields:
+                self.choices_fields = set()
+
+            # Don't bother testing if there's no `status` field.
+            if "status" not in self.choices_fields:
+                self.skipTest("Object does not contain a `status` field.")
+
+            # Save self.user as superuser to be able to view available choices on list views.
+            self.user.is_superuser = True
+            self.user.save()
```

Comment:
I feel like there should be an open issue here - why do we need to restrict the OPTIONS call to only superusers?

Reply:
It's only so that we don't have to worry about assigning permissions for each individual object on every pass, since the `testuser` has no permissions by default.